product_stock_state/README.rst

```diff
@@ -2,33 +2,49 @@
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
 
-========================
+===================
 Product Stock State
-========================
+===================
 
-This module add a stock state on the product in order to give an human readable information
-without giving the current quantity in stock
+This module add a stock state on the product in order to give an human readable
+information without giving the current quantity in stock.
 
-Installation
-============
-
-Nothing special required
+The state value can be :
 
+* In Stock
+* In Limited Stock (if qty available is under a threshold)
+* Resupplying (if qty forcasted is > 0)
+* Out of Stock (otherwise)
 
 Configuration
 =============
 
-Nothing required
+You can configure thresholds :
+
+* Globally, for a company, in the Sale Settings. It will be used for all
+  the products of the company.
+
+* On product category form. It will be used for all the products of this
+  category, or of the child categories. (User should be part of the new group
+  'Stock State Threshold by Category'.)
+
+* On product template form, for a specific product. (User should be part of
+  the new group 'Stock State Threshold by Product'.)
 
 Usage
 =====
 
-Go on the product tree/form and see the stock state
+Go on the product tree and see the stock state
+
+.. image:: /product_stock_state/static/description/product_product_tree.png
+     :width: 800 px
```

Comment:
TIL the real good way to define image in rst : image appears in githhub and in Odoo. Thanks you
/ + module_name
👍